CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Changelog
 
 ### Chores
 
+- Removed lint and flow errors and warnings ([PR 938](https://github.com/input-output-hk/daedalus/pull/938))
```

Comment:
That description does not really reflect what the PR does right? It `added flow checks and linting for storybook code` from what i see?